logcat/src/main/java/logcat/LogcatLogger.kt

```diff
@@ -28,56 +29,53 @@ interface LogcatLogger {
   )
 
   companion object {
+    val loggers: MutableList<LogcatLogger> = CopyOnWriteArrayList()
+
     @Volatile
     @PublishedApi
-    internal var logger: LogcatLogger = NoLog
+    internal var logExecutor: Executor? = null
       private set
 
     @Volatile
     private var installedThrowable: Throwable? = null
 
+    private val installLock = Any()
+
     val isInstalled: Boolean
       get() = installedThrowable != null
 
     /**
-     * Installs a [LogcatLogger].
+     * Installs the Logcat library, enabling logging. Logs will not be
```

Comment:
@pyricau I think you forgot someting here 😅

Reply:
Thank you, fixed!